book/src/building/swift-packages/README.md

```diff
@@ -53,159 +53,67 @@ Add a new `build.rs` file (`touch build.rs`):
 // my-rust-lib/build.rs
 
 use std::path::PathBuf;
+use swift_bridge_build::{GeneratePackageConfig, ApplePlatform};
 
 fn main() {
     let out_dir = PathBuf::from("./generated");
-
+    
     let bridges = vec!["src/lib.rs"];
     for path in &bridges {
         println!("cargo:rerun-if-changed={}", path);
     }
-
+    
     swift_bridge_build::parse_bridges(bridges)
         .write_all_concatenated(out_dir, env!("CARGO_PKG_NAME"));
 }
 ```
 
-Build the project for the desired platforms:
-
-```bash
-export SWIFT_BRIDGE_OUT_DIR="$(pwd)/generated"
-cargo build --target x86_64-apple-darwin
-cargo build --target aarch64-apple-ios
-cargo build --target x86_64-apple-ios
-```
-
-## Creating the XCFramework
-
-Go back to the root of the project and make a new directory `cd ..`.
-
-```bash
-mkdir MyFramework && cd $_
-```
-
-Copy the generated libraries and the headers to this folder:
-```bash
-mkdir include
-touch include/module.modulemap
-cp ../my-rust-lib/generated/SwiftBridgeCore.h ./include
-cp ../my-rust-lib/generated/my_rust_lib/my_rust_lib.h ./includ
-mkdir ios
-cp ../my-rust-lib/target/aarch64-apple-ios/debug/libmy_rust_lib.a ./ios
-mkdir macos
-cp ../my-rust-lib/target/x86_64-apple-darwin/debug/libmy_rust_lib.a ./macos
-mkdir simulator
-cp ../my-rust-lib/target/x86_64-apple-ios/debug/libmy_rust_lib.a ./simulator
-```
-
-This should result in the follwing folder structure:
-```
-MyFramework
-├── include
-│   ├── SwiftBridgeCore.h
-│   ├── module.modulemap
-│   └── my_rust_lib.h
-├── ios
-│   └── libmy_rust_lib.a
-├── macos
-│   └── libmy_rust_lib.a
-└── simulator
-    └── libmy_rust_lib.a
-```
+Create a new folder called *generated* `mkdir generated`.
 
-Edit `include/module.modulemap`:
+Create a new rust project called "post-build":
```

Comment:
This isn't a common pattern. So, instead of suggesting that people make a build crate we can just tell them that the generate package command exists and call it a day.

Something along the lines of:

---

After parsing your bridge modules and writing the bridging code to disk, you can take that bridging code and package it into a Swift Package.

First, build your Rust library for the desired platforms:

```bash
export SWIFT_BRIDGE_OUT_DIR="$(pwd)/generated"
cargo build --target x86_64-apple-darwin
cargo build --target aarch64-apple-ios
cargo build --target x86_64-apple-ios
```

Then use the `API` or `CLI` to package the bridging code and the Rust libraries into a Swift Package.

## API

```rust
use std::path::Path;
use std::collections::HashMap;
use swift_bridge_build::{GeneratePackageConfig, ApplePlatform};
fn main() {
    swift_bridge_build::generate_package(GeneratePackageConfig {
        bridge_dir: &Path::new("./generated"),
        paths: HashMap::from([
            (ApplePlatform::iOS, &"target/x86_64-apple-ios/debug/libmy_rust_lib.a" as _),
            (ApplePlatform::Simulator, &"target/aarch64-apple-ios/debug/libmy_rust_lib.a" as _),
            (ApplePlatform::macOS, &"target/x86_64-apple-darwin/debug/libmy_rust_lib.a" as _),
        ]),
        out_dir: &"MySwiftPackage",
        package_name: "MySwiftPackage"
    });
}
```

## CLI

Not yet implemented